src/cdk/platform/features/passive-listeners.ts

```diff
@@ -26,3 +26,14 @@ export function supportsPassiveEventListeners(): boolean {
 
   return supportsPassiveEvents;
 }
+
+/**
+ * Normalizes an `AddEventListener` object to something that can be passed
+ * to `addEventListener` on any browser, no matter whether it supports the
+ * `options` parameter.
+ * @param options Object to be normalized.
+ */
+export function normalizePassiveEventListenerOptions(options: AddEventListenerOptions):
```

Comment:
Add a unit test for this function specifically?

Reply:
That wouldn’t be very reliable, because the result depends on whether the browser supports passive events.

Reply:
Fair point